### PR TITLE
README.md: add an hello.js example

### PR DESCRIPTION
**Pull request by zoobab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,19 @@ cd jr
 make release.linux -j
 ```
 
+This is producing an executable named "jerry" in "build/bin/release.linux/".
```
Comment (akiss77) [inline README.md]: I'd merge this in the subsection below, e.g., "Once the executable `jerry` is built in `...` ...

File: README.md [hunk 2]
```diff
@@ -22,6 +22,19 @@ cd jr
...
+
+### Hello world
```
Comment (akiss77) [inline README.md]: Well, actually, it is not a hello _world_ :). "Running your First Script"? Or something like that.

File: README.md [hunk 3]
```diff
@@ -22,6 +22,19 @@ cd jr
...
+
+Once the executable "jerry" is built, you can try to launch the "hello.js" of the tests directory:
```
Comment (akiss77) [inline README.md]: I prefer backticks over quotes, since they stand out better when rendered. But that's not a strong opinion.

**Comment by zoobab** [comment]
@akiss77 as you wish for the syntax. It took me a while to find out how to make an "hello world", I think this is needed.

**Comment by akiss77** [comment]
@zoobab No question there (at least from my side). If you could update you patch, we could merge it in quite soon (I think/hope).

**Comment by LaszloLango** [comment]
@zoobab, I know that the current documentation must be improved, so any help with it is welcome. :) Please update your PR based on comments of @akiss77.

**Comment by LaszloLango** [comment]
@zoobab, please add your Sign-off to the commit message:
`JerryScript-DCO-1.0-Signed-off-by: Benjamin Henrion zoobab@gmail.com`

**Comment by akiss77** [comment]
The second patch in the PR seems to me as a somewhat unrelated change. This PR was about improving the docs, wasn't it? I'd prefer discussing test/example suite additions in a separate PR.
(BTW, copyright year is incorrect.)

**Comment by akiss77** [comment]
@zoobab You seem to use the branch associated with this PR as your working branch. I just realize that this is your master branch. Now, you have 13 patches here. That's not quite aligned with the traditional git/github workflow. Do you really want any of these changes get merged into the project?

**Pull request closed without merging**